src/dpos/components/ValidatorRewards.vue

```diff
@@ -1,4 +1,5 @@
 <template>
+  <main>
```

Comment:
There shouldn’t be more than on mains per page.

Reply:
changed now, ty